linera-chain/src/certificate.rs

```diff
@@ -129,7 +182,23 @@ impl From<ValidatedBlockCertificate> for Certificate {
     }
 }
 
-// TODO(#2842): In practice, it should be HashedCertificateValue = Hashed<CertificateValue>
+impl From<TimeoutCertificate> for Certificate {
+    fn from(cert: TimeoutCertificate) -> Certificate {
+        let TimeoutCertificate {
+            value,
+            round,
+            signatures,
+        } = cert;
+        let timeout = value.into_inner();
+        Certificate::new(
+            HashedCertificateValue::new_timeout(timeout.chain_id, timeout.height, timeout.epoch),
+            round,
+            signatures,
+        )
+    }
+}
+
+// In practice, it should be HashedCertificateValue = Hashed<CertificateValue>
```

Comment:
Shouldn't we still have this `TODO(#2842)`?

Reply:
Ah, I lost the line. But the issue is already addressed in the next PR https://github.com/linera-io/linera-protocol/pull/2858